messaging/types.go

```diff
@@ -9,6 +9,16 @@ import (
 	"github.com/status-im/status-go/wakuv2"
 )
 
+type Persistence interface {
+	WakuKeys() (map[string][]byte, error)
+	AddWakuKey(chatID string, key []byte) error
```

Comment:
Is it good to mention `Waku` here? Maybe `Transport` instead?

Reply:
Both are almost equal. Let's see how the API shapes up after extracting the remaining layers. Let's keep it as Waku for now, as Jazz proposed. Then we'll try to figure out the best naming.